app/models/bucket_list_adventure.rb

```diff
@@ -0,0 +1,6 @@
+class BucketListAdventure < ActiveRecord::Base
+  belongs_to :bucket_list
+  belongs_to :adventure
+
+  validates_inclusion_of :is_achieved, in: [true, false]
```

Comment:
Prefer the new style validations `validates :column, inclusion: value` over `validates_inclusion_of`.